src/Uno.UI/UI/Xaml/UIElement.debugging.cs

```diff
@@ -64,7 +64,7 @@ internal string GetThemeSource(string propertyName)
 				var output = "";
 				foreach (var binding in bindings)
 				{
-					output += $"{binding.ResourceKey} ({binding.Precedence}), ";
+					output += $"{binding.ResourceKey.Key} ({binding.Precedence}), ";
```

Comment:
Is this change part of the fix?

Reply:
It's a change to a debugging method, which I used to debug the bug, in order to arrive at the fix. So... yes 😋